lib/store_web/config.ex

```diff
@@ -14,6 +14,10 @@ defmodule StoreWeb.Config do
     adm_pages()
   end
 
+  def guest_pages() do
+    g_pages()
+  end
+
```

Comment:
Why do you need this extra level of indirection?

Reply:
Because i don't want to include orders.

Reply:
But why does the `g_pages` function exist? Why don't you just paste it there?